File: day012/day012-2.cs
using System;
class Day012 {
  static void Main() {
    
    int x, i, temp = 0;
    
    Console.Write("Girdiğiniz sayının tek veya çift olduğunu söyleyen programa hoş geldiniz! Lütfen bir sayı giriniz: ");
    x = Convert.ToInt32(Console.ReadLine());
    
    for (i = 1; i <= x/2 + 1; i += 1) {
        temp = i * 2;
        
        if (temp >= x){
            
            if (temp > x) {
                Console.WriteLine("Sayınız tek bir sayıdır.");
                break;
            }
            else {
                Console.WriteLine("Sayınız çift bir sayıdır.");
                break;
            }
        }
    }
  }
}

File: day012/day012-2.java
public class Day012
{
	public static void main(String[] args) {
		java.util.Scanner kb = new java.util.Scanner(System.in);
		int x, i, temp = 0;
		
		System.out.print("Girdiğiniz sayının tek veya çift olduğunu söyleyen programa hoş geldiniz! Lütfen bir sayı giriniz: ");
		x = kb.nextInt();
		
		for (i = 1; i <= x/2 + 1; i += 1) {
        temp = i * 2;
        
        if (temp >= x){
            
            if (temp > x) {
                System.out.println("Sayınız tek bir sayıdır.");
                break;
            }
            else {
                System.out.println("Sayınız çift bir sayıdır.");
                break;
            }
        }
    }
	}
}
